main.tf

```diff
@@ -822,6 +822,12 @@ module "prometheus-node-exporter" {
   target_repository = "${var.target_repository}/prometheus-node-exporter"
 }
 
+module "prometheus-pushgateway-bitnami" {
+  source = "./images/prometheus-pushgateway-bitnami"
+  # -bitnami is added in the module
+  target_repository = "${var.target_repository}/prometheus-pushgateway"
```

Comment:
This seems a little unusual, should this just have `-bitnami` here and not added at the end in the module?

Reply:
was following the example here https://github.com/chainguard-images/images/blob/main/main.tf#L870-L874